core/server/src/main/java/alluxio/worker/block/ClientRWLock.java

```diff
@@ -29,7 +29,7 @@
 public final class ClientRWLock implements ReadWriteLock {
   // TODO(bin): Make this const a configurable.
   /** Total number of permits. This value decides the max number of concurrent readers. */
-  private static final int MAX_AVAILABLE = 100;
+  private static final int MAX_AVAILABLE = 1000;
```

Comment:
I increased it so that my test would be able to read more than 100 times concurrently and I thought it was strange that there was such a limit. Do you want to keep it lower to avoid starvation?